### PR TITLE
Improve the JS WebHook template

### PR DESCRIPTION
**Pull request by davidebbo**

### CODE REVIEW

File: Templates/WebHook-Generic/index.js [hunk 1]
```diff
@@ -1,6 +1,16 @@
...
     context.log('Webhook was triggered!');
-    context.res = { body: 'WebHook processed successfully! \n' + JSON.stringify(data, null, " ") };
+
+    // Check if we got first/last properties
+    if("first" in data && "last" in data) {
+        context.res = { "body": {"greeting": 'Hello ' + data.first + " " + data.last + "!"}};
+    }
+    else {
+        context.res = {
+            "status": 400,
```
Comment (mathewc) [inline Templates/WebHook-Generic/index.js]: remove the extranous quotes from these properties (i.e. { status: 400, body: "..." })

Comment (davidebbo) [inline Templates/WebHook-Generic/index.js]: :+1:

File: Templates/WebHook-Generic/index.js [hunk 2]
```diff
@@ -1,6 +1,16 @@
 ﻿module.exports = function (context, data) {
-    context.res = {};
     context.log('Webhook was triggered!');
-    context.res = { body: 'WebHook processed successfully! \n' + JSON.stringify(data, null, " ") };
+
+    // Check if we got first/last properties
+    if("first" in data && "last" in data) {
+        context.res = { body: { greeting: 'Hello ' + data.first + " " + data.last + "!"}};
```
Comment (mathewc) [inline Templates/WebHook-Generic/index.js]: This would also be easier to read if it weren't all crammed on one line :) Rest looks good.

**Comment by paulbatum** [comment]
:shipit:

File: Templates/WebHook-Generic/index.js [hunk 3]
```diff
@@ -1,6 +1,16 @@
 ﻿module.exports = function (context, data) {
-    context.res = {};
     context.log('Webhook was triggered!');
-    context.res = { body: 'WebHook processed successfully! \n' + JSON.stringify(data, null, " ") };
+
+    // Check if we got first/last properties
+    if('first in data && 'last' in data) {
```
Comment (fabiocav) [inline Templates/WebHook-Generic/index.js]: Missing a quote here

Comment (paulbatum) [inline Templates/WebHook-Generic/index.js]: Nice catch!

**Comment by christopheranderson** [comment]
@davidebbo please fix the quote that @fabiocav pointed out and I'll merge

**Comment by davidebbo** [comment]
Merged.

**Pull request closed without merging**